app/javascript/packs/sdk.js

```diff
@@ -18,6 +18,7 @@ const runSDK = ({ baseUrl, websiteToken }) => {
     hasLoaded: false,
     hideMessageBubble: chatwootSettings.hideMessageBubble || false,
     isOpen: false,
+    isWidgetVisible: true,
```

Comment:
Can we use `hideMessageBubble` here. Isn't it the same thing?

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/4223/commits/5f7fa536ab1edd2413bb10d54d0b32cdc61b90cf